app/models/user.rb

```diff
@@ -6,7 +6,7 @@ class User < ApplicationRecord
   # @see https://github.com/attr-encrypted/attr_encrypted#usage
   #
   attr_encrypted :personal_access_token,
-                 key: Rails.application.secrets.encryption_key,
+                 key: Rails.application.credentials.personal_access_token_key,
```

Comment:
digging into it, the problem seems to lies with the fact that the key is a string in the credentials file, and needs to be a 32 byte array.  I've updated it so that it is saved as a 32 byte Hex string in the credentials file, then converted to a ByteArray in this line